packages/lesmis/src/components/EntityMenu.js

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
There is another story for the EntityMenu [here](https://app.zenhub.com/workspaces/active-sprints-5eea9d3de8519e0019186490/issues/beyondessential/tupaia-backlog/2496). This is just a placeholder